stockroom/external/importer/torchvision_importers.py

```diff
@@ -41,26 +41,61 @@ def variability_status(self):
     def __len__(self):
         return len(self.dataset)
 
+    @classmethod
+    def gen_splits(cls, torchvision_dataset, root):
+        dataset_splits = []
+        for train in [True, False]:
+            dataset = torchvision_dataset(root=root, train=train, download=True)
+            dataset_splits.append(cls(dataset, train))
+        return dataset_splits
+
 
 class Cifar10(TorchvisionCommon):
     name = 'cifar10'
 
-    def __init__(self, root, train=True):
-        dataset = datasets.CIFAR10(root=root, train=train, download=True)
-        super().__init__(dataset, train)
+    @classmethod
+    def gen_splits(cls, root):
+        dataset = datasets.CIFAR10
+        dataset_splits = super().gen_splits(dataset, root)
+        return dataset_splits
+
+    @staticmethod
+    def _process_data(img, lbl):
+        img = np.transpose(np.array(img), (2, 0, 1))
+        img = np.ascontiguousarray(np.array(img))
```

Comment:
You don't need the inner `np.array` in this line. It's already an `np.ndarray` object

Reply:
when I checked it `img` is a `pill.Image` object. Since we are overwriting the base method we have to convert it rt?
I tried to call the method in the base class but that seems not very neat since its a static method hence I thought of implementing it again.

Reply:
`img` is a `pil.Image` but that is converted to an np array on line 64. On line 65, you are calling `np.array` again

Reply:
Oh I didn't see that. My bad :sweat_smile: 
fixed it!